meta-rcar-gen3-xen/recipes-core/images/core-image-minimal.bbappend

```diff
@@ -1,10 +1,20 @@
 #Add Xen to build
 IMAGE_INSTALL_append = " \
     xen-base \
-    xen-flask \
```

Comment:
Flask is not being built by default. Its build is added in our development system by CONFIG_ passed through build command line.